recipes/qubekit/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "qubekit" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qubekit/QUBEKit/archive/v.{{ version }}.tar.gz
+  sha256: 61b940d34e74315da58fed624a134a4d39454c2cf21059b01b00f2aa71c0fed8
+
+
+build:
+  entry_points:
+    - QUBEKit = QUBEKit.run:main
+    - qubekit = QUBEKit.run:main
+    - QUBEKit-pro = QUBEKit.proteins.protein_run:main
+    - QUBEKit-gui = QUBEKit.GUI.gui:main
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
```

Comment:
You don't need steuptools if you have pip

Reply:
thanks removed!